docs/release_process.md

```diff
@@ -146,55 +146,94 @@ After the release branches are created, preliminary testing should be conducted
 After preliminary testing is complete, project reference updates should be made to ensure that all projects are referencing the correct versions of other projects. Once project references are updated, the release should be created by merging the `release_(year)-(quarter)` branch into the `master` branch and tagging the release with the appropriate version number. Images should be built and pushed to DockerHub for testing.
 
 ### Steps
-#### Merging release branches & updating project references
+#### Merging release branches, updating project references and creating releases
 1. Merge ‘release_(year)-(quarter)’ branch into ‘master/main’ branch for the following projects:
     - asn1_codec
     - jpo-cvdp
     - jpo-security-svcs
     - jpo-sdw-depositor
     - jpo-s3-deposit
 
-    1a. Tag the master/main branch of each application with a git tag that includes the version number of each app.
+    1a. Create a release for the asn1_codec project from the ‘master/main’ branch and tag the release with the version number of the release. (e.g. asn1_codec-x.x.x)
 
-2. Update git submodule references for the ‘jpo-ode’ project to point to tagged commits in projects with updated `master/main` branches. Also update the version numbers within the pom.xmls of each of the ode subprojects (jpo-ode-common, jpo-ode-plugins, jpo-ode-svcs) to be self-consistent.
+    1b. Create a release for the jpo-cvdp project from the ‘master/main’ branch and tag the release with the version number of the release. (e.g. jpo-cvdp-x.x.x)
     
-    2a. (These changes will need to pass CI/CD checks & make it into the `release_(year)-(quarter)` branch before continuing.)
-
-3. Merge `release_(year)-(quarter)` branch into `master/main` branch for the jpo-ode project, and add a git tag with the ode version number.
+    1c. Create a release for the jpo-security-svcs project from the ‘master/main’ branch and tag the release with the version number of the release. (e.g. jpo-security-svcs-x.x.x)
+    
+    1d. Create a release for the jpo-sdw-depositor project from the ‘master/main’ branch and tag the release with the version number of the release. (e.g. jpo-sdw-depositor-x.x.x)
+    
+    1e. Create a release for the jpo-s3-deposit project from the ‘master/main’ branch and tag the release with the version number of the release. (e.g. jpo-s3-deposit-x.x.x)
 
-4. Update git submodule references for the ‘jpo-geojsonconverter’ project to point to the tagged commit in jpo-ode master/main branch.
+2. Update git submodule references for the ‘jpo-ode’ project to point to tagged commits in projects with updated `master/main` branches. Also update the version numbers within the pom.xmls of each of the ode subprojects (jpo-ode-common, jpo-ode-plugins, jpo-ode-svcs) to be self-consistent.
```

Comment:
Maybe add explicit steps to update both the `<version>` and the dependencies in pom's of the ode subprojects for example:
`jpo-ode-plugins` dependency on `jpo-ode-common`
`jpo-ode-svcs` dependencies on `jpo-ode-core` and `jpo-ode-plugins`

Reply:
This is now a separate, explicit step with substeps.